cmd/bosun/web/static/partials/config.html

```diff
@@ -33,18 +32,90 @@
 	color: lightgray;
 }
 .lines .lineerror {
-	color: #a94442;
-	background: #f2dede;
+	color: black;
+	font-weight: bold;
+	background-color: lightgray;
 }
+
 </style>
+<label style="float:left; margin-right:15px;">Jump to:</label>
+<div class='row'>
+<div class="dropdown" ng-repeat="(type,list) in items" style="float:left; margin-right:5px;margin-bottom:15px;">
+  <button class="btn btn-primary dropdown-toggle" type="button" id="itemSelector" data-toggle="dropdown" aria-expanded="true">
+ 	{{type}}
+    <span class="caret"></span>
+  </button>
+  <ul class="dropdown-menu" role="menu" aria-labelledby="itemSelector">
```

Comment:
aria is the accessibility stuff (for blind folks)

On Wed, Apr 15, 2015 at 12:38 PM Craig Peterson <notifications@github.com>
wrote:

> In cmd/bosun/web/static/partials/config.html
> <https://github.com/bosun-monitor/bosun/pull/868#discussion_r28437553>:
>
> >  </style>
> > +<label style="float:left; margin-right:15px;">Jump to:</label>
> > +<div class='row'>
> > +<div class="dropdown" ng-repeat="(type,list) in items" style="float:left; margin-right:5px;margin-bottom:15px;">
> > +  <button class="btn btn-primary dropdown-toggle" type="button" id="itemSelector" data-toggle="dropdown" aria-expanded="true">
> > + 	{{type}}
> > +    <span class="caret"></span>
> > +  </button>
> > +  <ul class="dropdown-menu" role="menu" aria-labelledby="itemSelector">
>
> what is aria? I just pasted the bootstrap snippet. Not confidence
> inspiring I know.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/bosun-monitor/bosun/pull/868/files#r28437553>.
>